### PR TITLE
added Ritwik880

### LINKED ISSUE #627: add Ritwik880
Screenshot (345)

### PR DESCRIPTION
**Pull request by Ritwik880**
Resolve #627

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

File: data.json [hunk 1]
```diff
@@ -1204,5 +1204,11 @@
     "githubUsername": "SahityaRoy",
     "imageUrl": "https://user-images.githubusercontent.com/72821604/127466372-6f894759-7511-4bd1-828c-30734755102e.png",
     "issueNumber": 621
+  },
+  {
+    "name": "Ritwik Sinha",
+    "githubUsername": "Ritwik880",
+    "imageUrl": "https://user-images.githubusercontent.com/54446880/127509715-b594b68d-4703-4bc2-aed5-1172d7df6a5c.png",
+    "issueNumber": 621
```
Comment (Panquesito7) [inline data.json:1212]: Wrong issue number.

```suggestion
    "issueNumber": 627
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Ritwik880** [comment]
Hey @Panquesito7 now I have updated.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 😄

**Review by w3bdesign — APPROVED** [review]

File: data.json (hunk 1 above)
Comment (schmelto) [reply data.json]: Good catch!

**Comment by schmelto** [comment]
Hey @Ritwik880 could you resolve the merge conflict. If you need any halp please let us know.

**Comment by Ritwik880** [comment]
Hey @schmelto I think now everything is fine, please have a look.

File: data.json [hunk 2]
```diff
@@ -1205,10 +1205,18 @@
     "imageUrl": "https://user-images.githubusercontent.com/72821604/127466372-6f894759-7511-4bd1-828c-30734755102e.png",
     "issueNumber": 621
   },
+  {
+
```
Comment (schmelto) [inline data.json:1209]:
```suggestion
```

File: data.json [hunk 3]
```diff
@@ -1205,10 +1205,18 @@
...
+    "githubUsername": "Ritwik880",
+    "imageUrl": "https://user-images.githubusercontent.com/54446880/127509715-b594b68d-4703-4bc2-aed5-1172d7df6a5c.png",
+    "issueNumber": 627
+  },
   {
     "name": "Sri Subathra Devi B",
     "githubUsername": "Subathra19",
     "imageUrl": "https://user-images.githubusercontent.com/55084520/127505296-cd9fe349-ed69-4ff8-ad10-4b363cdf8f34.PNG",
     "issueNumber": 625
+
```
Comment (schmelto) [inline data.json:1220]:
```suggestion
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json [hunk 4]
```diff
@@ -1205,10 +1205,19 @@
     "imageUrl": "https://user-images.githubusercontent.com/72821604/127466372-6f894759-7511-4bd1-828c-30734755102e.png",
     "issueNumber": 621
   },
+
```
Comment (schmelto) [inline data.json:1208]:
```suggestion
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by schmelto** [comment]
Those tree empty lines should not be there

**Comment by Ritwik880** [comment]
Hey @schmelto I am not getting your point??

Screenshot (348)

**Comment by schmelto** [comment]
Line 1214
Line 1208

Those you can remove due those are empty

**Comment by Ritwik880** [comment]
Hey @schmelto I think now its fine.

**Review by AvidCoder101 — APPROVED** [review]
LGTM @schmelto or @kaiwalyakoparkar this PR can be merged I think

**Review by Panquesito7 — APPROVED** [review]

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for your collaboration and contribution

**Pull request merged**